EKF/ekf.cpp

```diff
@@ -359,6 +364,10 @@ void Ekf::predictState()
 	// filter and limit input between -50% and +100% of nominal value
 	input = math::constrain(input, 0.5f * FILTER_UPDATE_PERIOD_S, 2.0f * FILTER_UPDATE_PERIOD_S);
 	_dt_ekf_avg = 0.99f * _dt_ekf_avg + 0.01f * input;
+
+	// some calculations elsewhere in code require a raw angular rate vector so calculate here to avoid duplication
+	_ang_rate_delayed_raw = _imu_sample_delayed.delta_ang / fmaxf(_imu_sample_delayed.delta_ang_dt, FILTER_UPDATE_PERIOD_S / 4);
```

Comment:
Is there a reason for lower bounding the dt? Why `FILTER_UPDATE_PERIOD_S / 4`?

Reply:
Zero is bad obviously, so  /4 was chosen to give plenty of margin for frame jitter.

Reply:
Is it possible for delta_ang_dt to become 0?